packages/frontend/src/utils/formatters/formatTimeLeft.ts

```diff
@@ -12,7 +12,7 @@ export function formatTimeLeft(timestamp?: BigNumber, now = Date.now()) {
   const difference = date.diff(now) > 0 ? date.diff(now) : 0
   const duration = moment.duration(difference)
 
-  const days = padZeroes(duration.asDays().toFixed(0))
+  const days = padZeroes(duration.days())
```

Comment:
I think `asDays()` has been used because `moment.prototype.days()` returns the number of days left over from full months (as in if `duration` is 35 days, `duration.days()` will return 5). I guess to make it accurate, we should use `Math.floor(duration.asDays())`